nltk/tgrep.py

```diff
@@ -0,0 +1,842 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
@wroberts are you happy to add our standard header with copyright owned by the NLTK project please?

```
# Natural Language Toolkit: tgrep
#
# Copyright (C) 2001-2015 NLTK Project
# Author: Will Roberts <email>
# URL: <http://nltk.org/>
# For license information, see LICENSE.TXT
```